### PR TITLE
Discrepancy between smtplib.SMTP and pyramid_mailer configurations.

### PR DESCRIPTION
**Pull request by maparent**
Converge on pyramid_mailer.

### CODE REVIEW

File: assembl/models/notification.py [hunk 1]
```diff
@@ -967,40 +968,43 @@ def get_to_email_address(self):
...
-        return msg.as_string()
+        headers['List-Subscribe'] = frontendUrls.getUserNotificationSubscriptionsConfigurationUrl()
+        headers['List-Unsubscribe'] = frontendUrls.getUserNotificationSubscriptionsConfigurationUrl()
+
+        subject = Header(self.get_notification_subject(), 'utf-8')
+
+        sender = Header(self.event_source_object().creator.name, 'utf-8')
+        sender.append(" <" + self.get_from_email_address() + ">", 'ascii')
+        recipient = self.get_to_email_address()
+        message = Message(
```
Comment (ayyazdaniaryan) [inline assembl/models/notification.py]: Ah okay, you changed the structure from `msg` object to `headers` to comply with pyramid mailer?

File: assembl/models/notification.py [hunk 2]
```diff
@@ -967,40 +968,43 @@ def get_to_email_address(self):
...
+            subject=subject.encode(),
+            sender=sender.encode(),
```
Comment (ayyazdaniaryan) [inline assembl/models/notification.py]: the encode method encodes what format to what? Would there be any issues with the unicode strings of multi-language texts?

File: assembl/views/api2/notification.py [hunk 3]
```diff
@@ -85,7 +85,7 @@ def mail_text_preview(request):
              ctx_instance_class=Notification, permission=P_READ,
              accept="text/html", name="mail")
 def mail(request):
-    return Response(request.context._instance.render_to_email(),
+    return Response(request.context._instance.render_to_message().to_message().as_string(),
```
Comment (ayyazdaniaryan) [inline assembl/views/api2/notification.py]: `as_string` returning plain text or html?

File: assembl/models/notification.py (hunk 2 above)
Comment (maparent) [inline assembl/models/notification.py]: precisely for encoding. to quoted-printable.

**Pull request closed without merging**